packages/frontend/src/components/EpochView.jsx

```diff
@@ -54,116 +44,23 @@ export default observer(({ deployment }) => {
         />
       </div>
 
-      {/* Activity By Epoch graph header */}
-      {/* <div className="flex">
-        <h3>Activities (Epoch {selectedEpochActivities})</h3>
-        <div className="dropdown">
-          <button className="dropbtn">Jump to epoch..</button>
-          <div className="dropdown-content">
-            {attester.epochsByAttester.map(({ number }) => (
-              <p
-                onClick={() => setSelectedEpochActivities(number)}
-                key={number}
-              >
-                {number}
-              </p>
-            ))}
-          </div>
-        </div>
-      </div> */}
-
-      {/* data for Activity By Epoch graph element */}
-      {/* <div className="graph-container" style={{ height: 'auto' }}>
-        <ul>
-          SIGNUPS THIS EPOCH: {signups ? signups.length : 0}
-          {signups
-            ? signups.map(({ commitment, timestamp }) => (
-                <li key={commitment}>
-                  {dayjs(timestamp * 1000).format('MM-DD-YYYY HH:mm:ss')}
-                </li>
-              ))
-            : null}
-          {signups ? null : <li>There were no signups in this epoch</li>}
-        </ul>
-        <ul>
-          ATTESTATIONS THIS EPOCH:{' '}
-          {graphAttestations ? graphAttestations.length : 0}
-          {graphAttestations
-            ? graphAttestations.map(({ epochKey, timestamp }) => (
-                <li key={epochKey}>
-                  {dayjs(timestamp * 1000).format('MM-DD-YYYY HH:mm:ss')}
-                </li>
-              ))
-            : null}
-          {graphAttestations ? null : (
-            <li>There were no attestations in this epoch</li>
-          )}
-        </ul>
-        <ul>
-          UST THIS EPOCH: {USTs ? USTs.length : 0}
-          {USTs
-            ? USTs.map(({ nullifier, timestamp }) => (
-                <li key={nullifier}>
-                  {dayjs(timestamp * 1000).format('MM-DD-YYYY HH:mm:ss')}
-                </li>
-              ))
-            : null}
-          {USTs ? null : <li>There were no USTs in this epoch</li>}
-        </ul>
-      </div> */}
-
       <div className="flex">
-        <h3>Attestations (Epoch {selectedEpochAttestations})</h3>
-        <div className="dropdown">
-          <button className="dropbtn">Jump to epoch..</button>
-          <div className="dropdown-content">
-            {attester.epochsByAttester.map(({ number }) => (
-              <p
-                onClick={() => setSelectedEpochAttestations(number)}
-                key={number}
-              >
-                {number}
-              </p>
-            ))}
-          </div>
-        </div>
+        <h3>Attestations</h3>
```

Comment:
If we are getting rid of the feature for viewing an attester's data per epoch, the `AttesterPage` layout really doesn't make sense anymore. bc now we're not showing any data per epoch except the epoch# and transition time.
Maybe these 2 boxes should go right under the 4 boxes at the top, and instead of `Epoch / Users`, we toggle `Attestations / Users`.

<img width="896" alt="Screen Shot 2023-02-22 at 1 51 59 PM" src="https://user-images.githubusercontent.com/79813968/220756830-faab0956-90c3-411d-a1b0-6767a0dba8fa.png">

also if the user is not selecting an epoch, probably we want to add epoch# and timestamp to the attestation row?

Reply:
Agreed, c12a4b6

Reply:
yes, but I meant the `Current Epoch` and `Epoch Transition Time` should come out of the toggled view and stay on the main page under the other attester info boxes. Probably more UI than you want to do :)
agreed? I can handle this with the other UI stuff.

Reply:
Ah, I understand. The `calculateCurrentEpoch` function should be moved out of the `EpochView` component and into maybe the attester store so it can be reused.

Right now i'm not super concerned with this ui quirk. I really want to push this update so i can give it to the taz team. This ui is mostly a dev tool right now, there's plenty of time to clean it up before normal users see it.

Reply:
ok. cool if I take until tomorrow this time-ish to finish up?

Reply:
Yes, i think the contents of this pr are good enough to distribute to other teams

Reply:
Yep, all good